_includes/home/connect.md

```diff
@@ -50,7 +50,7 @@ When you build on Firefox, you build with a community of extension developers. A
 
 [Learn more about the community](https://wiki.mozilla.org/Add-ons/Community/)
```

Comment:
Sorry I missed this earlier. This needs to link to `https://extensionworkshop.com/community` (in the code you should be able to shorten this to `/community/`.